Dvorak-Right-Handed/KeyboardView.swift

```diff
@@ -159,8 +172,12 @@ class KeyboardView: UIView {
         symbols = !symbols
         if symbols {
             self.symKey.setTitle("ABC" , for: .normal)
-            let _ = keys.map({$0.setTitle(Keys.symKeys[($0.restorationIdentifier ?? "☂︎")], for: .normal)})
-        } else {
+            if self.nibName.starts(with: "Full") {
+                let _ = keys.map({$0.setTitle(Keys.symKeysFull[($0.restorationIdentifier ?? "☂︎")], for: .normal)})
```

Comment:
It's not clear where "Keys" is coming from. I guess it is coming from the Keys.swift file but that is not obvious. Not sure how to fix this.
Edit: So you changed Keys.swift to KeysModel.swift. Consider using KeysModel.Keys instead of just Keys.

Reply:
yes, I did not make it explicit before; I will fix.